cmd/lncli/commands_test.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseChannelPointStr(t *testing.T) {
```

Comment:
nit: add a godoc describing the test `// TestParseChannelPointStr tests ...`

Reply:
added